pool.go

```diff
@@ -235,7 +239,20 @@ type ScalingPolicy interface {
 // DefaultScalingPolicy returns a new instance of the default scaling policy.
 // Instances returned by this function should not be reused.
 func DefaultScalingPolicy() ScalingPolicy {
-	return MovingAverage(10, MinMax(1, 10, AIMD(1, 0.5)))
+	maxInstances := 0
+	// Try to read maxInstances from the environment variable
+	maxInstancesEnv := os.Getenv("BBLFSH_SCALING_MAX_INSTANCES")
```

Comment:
I think it'd be better to move this context dependant code outside and pass them as arguments explicitely